app/views/spree/products/_gallery.html.erb

```diff
@@ -1,25 +1,46 @@
-<div class="col-12 col-md-7 col-lg-7" data-hook="product_left_part">
-  <% if @product_images.size > 0 %>
+<!-- <div class="col-12 col-md-7 col-lg-7" data-hook="product_left_part">
+  <%# if @product_images.size > 0 %>
     <div
       class="w-100 d-flex align-self-start justify-content-between product-details-images"
       data-hook="product_left_part_wrap">
 
       <div class="d-none d-md-block product-details-thumbnails" data-hook="product_thumbnails">
-        <%= render 'spree/shared/carousel/thumbnails',
+        <%#= render 'spree/shared/carousel/thumbnails',
                   images: @product_images,
                   group_id: 'main-product-carousel',
                   id: 'productThumbnailsCarousel',
                   trigger_id: 'productCarousel',
                   per_page: 5 %>
       </div>
       <div class="product-details-single" data-hook="product_images">
-        <%= render 'spree/shared/carousel/single',
+        <%#= render 'spree/shared/carousel/single',
                   images: @product_images,
                   group_id: 'main-product-carousel',
                   id: 'productCarousel',
                   trigger_id: 'productCarousel',
                   zoomed: false %>
       </div>
     </div>
-  <% end %>
+  <%# end %>
+</div> -->
+<% if (@product.images + @product.variant_images).uniq.size > 1 %>
+<div class="col-xl-7" data-hook="product_left_part" data-hook="product_left_part_wrap">
+  <div class="product-gallery-wrap" data-hook="product_thumbnails">
+    <ul id="product-gallery"  data-hook data-hook="product_images">
+      <% @product.images.each do |i| %>
+        <li data-thumb= <%= main_app.url_for(i.url(:product)) %> <%= i.viewable.id %> >
+          <%= image_tag(main_app.url_for(i.url(:large))) %>
+        </li>
+      <% end %>
+      <% if @product.has_variants? %>
+        <% @product.variant_images.each do |i| %>
+          <% next if @product.images.include?(i) %>
+          <li <%= i.viewable.id %>>
```

Comment:
Why we didnt added this data hook (data-thumb= <%= main_app.url_for(i.url(:product))) to this <li> . As i can see its added in above code